src/main.js

```diff
@@ -188,11 +188,26 @@ module.exports = {
       scope: 'file',
       lintsOnChange: true,
       lint: async (textEditor) => {
-        const text = textEditor.getText()
-        if (text.length === 0) {
-          return []
```

Comment:
__*Question*__

Did you mean to remove this check?

Reply:
Hmmm, nope. I _meant_ to move it down below the path checks along with the `getText` call. 🐛

Reply:
Oh, actually, yes I did mean to remove it. With it in there we are saying that an empty file is completely valid, but we have _no_ idea what a user's configuration is, or what plugins they have or rules they have enabled. We have no way of knowing whether an empty file is considered "valid" under their configuration so we should remove this optimization and do the full ESLint run even on an empty file.